### PR TITLE
ci: add iot-simplified test case workflow

### PR DESCRIPTION
**Pull request by yih-redhat**
Add test workflow for iot-simplified-installer on f39

### CODE REVIEW

File: .github/workflows/fedora-39.yml [hunk 1]
```diff
@@ -446,3 +447,81 @@ jobs:
...
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Clone repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.pr-info.outputs.sha }}
+          fetch-depth: 0
+
+      - name: run ostree-simplified-installer.sh
+        run: ./ostree-simplified-installer.sh
```
Comment (henrywang) [inline .github/workflows/fedora-39.yml:497]: You should define env var DOWNLOAD_NODE here.

**Review by henrywang — CHANGES_REQUESTED** [review]
Thanks @yih-redhat. I have one comment.

File: .github/workflows/fedora-39.yml (hunk 1 above)
Comment (yih-redhat) [reply .github/workflows/fedora-39.yml:497]: I thought fedora test does not require this env, for example, ostree.sh test does not have it, https://github.com/virt-s1/rhel-edge/blob/main/.github/workflows/fedora-39.yml#L93
Did I miss anything here?

Comment (henrywang) [reply .github/workflows/fedora-39.yml:497]: Oh, right. simplified installer test script does not need DOWNLOAD_NODE.

**Review by henrywang — APPROVED** [review]

**Pull request merged**